docs/source/changelog.rst

```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+0.4.6 - 2022-01-11
+------------------
```

Comment:
```suggestion
Unreleased
----------
```

This new section should just be named 'unreleased', and will be updated when the next release of the library is created.